test/message/test_plan.py

```diff
@@ -420,9 +421,9 @@ def forward(self, x):
 
 def test_replace_worker_ids_two_strings(hook):
     plan = sy.Plan(id="0", owner=hook.local_worker, name="test_plan")
-    _replace_message_ids_orig = sy.federated.Plan._replace_message_ids
+    _replace_message_ids_orig = sy.msg.Plan._replace_message_ids
```

Comment:
Could you unify the way the import is done? (sy.Message vs sy.msg.Message vs sy.msg.Plan)
sy.msg.Plan => msg.Plan

Reply:
done